assets/sass/components/settings/_googlesitekit-settings-user-input.scss

```
@@ -0,0 +1,24 @@
+/**
+ * Settings user input styles.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-settings-user-input {
+
+	.googlesitekit-user-input__preview {
+		padding: 25px !important;
+	}
+}
```

Just start your rule from `.googlesitekit-plugin` and you won't need `!important`:

```suggestion
.googlesitekit-plugin {
    .googlesitekit-settings-user-input {
        .googlesitekit-user-input__preview {
            padding: 25px;
        }
    }
}
```